pkg/providers/cloudstack/decoder/decoder.go

```diff
@@ -61,6 +62,21 @@ func ParseCloudStackSecret() (*CloudStackExecConfig, error) {
 	}, nil
 }
 
+func GetCloudStackExtraMounts() ([]string, error) {
+	var extraMounts []string
+	env, found := os.LookupEnv(EksaCloudStackHostPathToMount)
```

Comment:
it seems like this should be a list object on the cloudstackdatacenterconfig, no? Could you remind me once more why we decided to go with the env var route?

Reply:
Check this @vivek-koppuru 's comment about the env var approach: https://github.com/aws/eks-anywhere/pull/1981#issuecomment-1122626761